packages/smooth_app/lib/main.dart

```diff
@@ -170,8 +175,55 @@ class SmoothAppGetLanguage extends StatelessWidget {
 
   final Widget appWidget;
 
+  Future<void> _initOpenFoodFactsDart(BuildContext context) async {
+    //We can't check the credentials if the user never logged in
+    if (!await UserManagementHelper.checkCredentialsInStorage()) {
+      return;
+    }
+
+    final AppLocalizations appLocalizations = AppLocalizations.of(context)!;
+    final bool stillCorrect = await UserManagementHelper.checkAndReMountCreds();
+
+    if (!stillCorrect) {
+      showDialog<dynamic>(
+        context: context,
+        builder: (final BuildContext context) => SmoothAlertDialog(
+          close: true,
```

Comment:
What if I close the dialog: nothing happens, right?

Reply:
Yes nothing, this `close: true,`  just adds a cross on the top right